service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/CreateAiNotebookInstanceStep.java

```diff
@@ -55,7 +56,9 @@ public class CreateAiNotebookInstanceStep implements Step {
 
   /** Default post-startup-script when starting a notebook instance. */
   protected static final String DEFAULT_POST_STARTUP_SCRIPT =
-      "https://raw.githubusercontent.com/DataBiosphere/terra-workspace-manager/main/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh";
+      "https://raw.githubusercontent.com/DataBiosphere/terra-workspace-manager/%s/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh";
```

Comment:
This seems like a very specific thing to be implemented inside of WSM. I think it should be a property in `application.yml`. It can be set to something other than the default by setting an environment variable.

Reply:
the branch hash is already a property in the application.yml though.

Reply:
I'm asking why the path to the startup script has to be hardcoded here?

Reply:
well :) that was some tech debt.